app/src/main/AndroidManifest.xml

```diff
@@ -6,15 +6,18 @@
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.PACKAGE_USAGE_STATS" />
+    <uses-permission android:name="android.permission.INTERNET" />
 
     <application
         android:allowBackup="false"
         android:icon="@mipmap/openlattice_logo"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/openlattice_logo_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true">
```

Comment:
This looks like a bad idea. It seems this enables HTTP traffic, which we want to avoid. What's the use case for this?

Reply:
This was necessary for dev since dev server is http